packages/govtnz-ds/src/normalize-govuk.ts

```diff
@@ -166,11 +166,29 @@ export const govukToMetaTemplateInput = async (
       html = `<a class="g-back-link" href="#"><mt-variable key="children">Example text</mt-variable></a>`;
       break;
     }
-    case 'button__disabled': {
+
+    case 'button':
+    case 'button__disabled':
+    case 'button__with-active-state':
+    case 'button__with-focus-state':
+    case 'button__with-hover-state':
+    case 'button__secondary':
+    case 'button__warning': {
       id = 'button';
-      html = `<button class="g-button {{ disabled!?: g-button--disabled }}" type="submit" aria-disabled="{{ disabled!?: true }}" disabled="{{ disabled!?: true }}"><mt-variable key="children">Example text</mt-variable></button>`;
+      html = 
+        `<button
+          class="g-button {{ disabled!?: g-button--disabled }} {{ kind: g-button--secondary as secondary | g-button--warning as warning }}"
```

Comment:
hmm... I wonder about "kind" as a name, but naming things is always hard! Any alternatives that come to mind? I wonder how other DSs explain this category of button. We don't want to use "type" of course because buttons already have that as an attribute.

Reply:
As discussed offline - let's go with `level` for now.